action.yml

```diff
@@ -0,0 +1,14 @@
+# action.yml
+name: 'Check DCO Signoff'
+description: 'A reusable action that checks if all commit has a DCO signoff'
+runs:
+  using: "composite"
+  steps:
+    - name: Set up Python
+      uses: actions/setup-python@v4
```

Comment:
You could use `@v5` https://github.com/actions/setup-python?tab=readme-ov-file#basic-usage